eslint-example/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+		"globals": {"chrome": true},
+    "rules": {
+        "no-set-state": "off"
+    },
+    "env": {
+    		"browser": true,
+        "es6": true
```

Comment:
Ah, yes! From the documentation, this appears to be Firefox compatible (everything but imports).